templates/site_index.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'base.html' %}
 {% load mezzanine_tags wagtailcore_tags %}
 
-{% block page-title %}{% if page %}{{ page.meta_title }}{% else %}The Center for Digital Humanities at Princeton{% endif %}{% endblock %}
+{% block page-title %}Home{% endblock %}
```

Comment:
What's the difference between this template and the cdhpages home page template? Is this one no longer in use?

Reply:
yup – I edited both for consistency but this one won't be used anymore.